ui/scss/component/_channel.scss

```diff
@@ -267,6 +267,12 @@ $actions-z-index: 2;
   margin-right: var(--spacing-xs);
 }
 
+.channel-thumbnail--xxsmall {
+  height: 16px;
+  width: 16px;
+  margin-right: var(--spacing-s) !important;
```

Comment:
With reference to it's peers (`.channel-thumbnail--xsmall`, etc.), would `1rem` be more appropriate so that everything scales accordingly when we change the base in the future?

Side comment (no changes needed, just a thought):  
The increasing amount of `!important` usages lately feels [concerning](https://uxengineer.com/css-specificity-avoid-important-css/), especially when used against our own css.
- If I used `.channel-thumbnail--xxsmall` in new component, would that `margin-right` make sense as the default?
- Would `.channel-selector .channel-thumbnail--xxsmall` be a better method than `!important`, as it conveys why the override was needed? (I'm just guessing on the reason for `!important`).

Reply:
agreed